README.md

```diff
@@ -24,93 +37,127 @@ Add two new maven repository destination in the project-level build.gradle repos
+}
 ```
-2.1.2. To get callbacks via the background service then one would have to create a service which  extends **VeriffStatusUpdatesService**. Overwriting the method **onStatusChanged** will enable the developer to listen for different statuses coming from the Veriff SDK
+
+### Using a background service
+
+To capture the result from the SDK using a background service.
+
 ```java
-public class VeriffSDKStatusUpdatesService extends VeriffStatusUpdatesService {
+public class MyVeriffStatusUpdatesService extends VeriffStatusUpdatesService {
    @Override
    protected void onStatusChanged(String sessionToken, int statusCode) {
-       if (statusCode == VeriffConstants.STATUS_USER_FINISHED) {
-           //user finished whatever he/she was asked to do, there might be other callbacks coming after this one (for example if the images are still being uploaded in the background)
-       } else if (statusCode == VeriffConstants.STATUS_ERROR_NO_IDENTIFICATION_METHODS_AVAILABLE) {
-           //there are no identifications methods currently available
-       } else if (statusCode == VeriffConstants.STATUS_ERROR_SETUP) {
-           //issue with the provided vendor data
-       } else if (statusCode == VeriffConstants.STATUS_ERROR_UNKNOWN) {
-           //unidentified error
-       } else if (statusCode == VeriffConstants.STATUS_ERROR_NETWORK) {
-           //network unavailable
-       } else if (statusCode == VeriffConstants.STATUS_USER_CANCELED) {
-           //user closed SDK
-       } else if (statusCode == VeriffConstants.STATUS_UNABLE_TO_ACCESS_CAMERA) {
-           //we are unable to access phone camera (either access denied or there are no usable cameras)
-       } else if (statusCode == VeriffConstants.STATUS_UNABLE_TO_RECORD_AUDIO) {
-           //we are unable to access phone microphone
-       } else if (statusCode == VeriffConstants.STATUS_SUBMITTED) {
-           //SelfID photos were successfully uploaded
-       } else if (statusCode == VeriffConstants.STATUS_OUT_OF_BUSINESS_HOURS) {
-           //call was made out of business hours, there were no verification specialists to handle the request
-       } else if (statusCode == VeriffConstants.STATUS_ERROR_SESSION) {
-           //invalid sessionToken was passed to the SDK
-       } else if (statusCode == VeriffConstants.STATUS_DONE) {
-           //verification specialist declined the session
-       } else if (statusCode == VeriffConstants.STATUS_VIDEO_CALL_ENDED) {
-           //video call flow was finished
-       }
+       handleResult(statusCode, sessionToken); //see below to know how to handle result
    }
 }
 ```
 
-  After the service is set up then it needs to be declared in the **application manifest**:
-```
+Add the service to the application manifest:
+
+```xml
 <!-- Implement Veriff SDK updates service -->
-<service android:name=".service.VeriffSDKStatusUpdatesService">
+<service android:name=".MyVeriffStatusUpdatesService">
```

Comment:
unrelated to this commit, but oof, doesn't this export the service? :scream:

Reply:
Yes it will make it exported.

Reply:
Its exported=false in the sample app manifest LOL

Reply:
no need to change it here, I'll file a ticket in JIRA